src/v3/src/components/Form/SimpleLayout.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Box } from '@mui/material';
+import { FunctionComponent, h } from 'preact';
+
+import {
+  UISchemaElement,
+  UISchemaElementComponent,
+  UISchemaLayout,
+  UISchemaLayoutType,
+} from '../../types';
+import { getElementKey, isDevelopmentEnvironment, isTestEnvironment } from '../../util';
+import renderers from './renderers';
+
+/**
+ *
+ * The purpose of this component is to act as an abstracted layer
+ * for use by non standard UISchemaLayout interfaces to prevent a circular dependency
+ * i.e. {@link AccordionLayout} and {@link StepperLayout}
+ *
+ */
+const LayoutContainer: FunctionComponent<{ uischema: UISchemaLayout }> = ({ uischema }) => {
```

Comment:
This component is basically a duplicate of `Layout.tsx` but without the dependency on `Stepper`, or `Accordion` (as its purpose is to just render standard UISchema Layout and elements.